tests-mainnet/Strategy/yearn_vault/valide_data.json

```diff
@@ -0,0 +1,25 @@
+{
+    "test_parameter": [
```

Comment:
Why do we have 2 parameter files?

Reply:
for each case I've created 2 parameters because the fied number are differents .